html/ajax_listports.php

```diff
@@ -20,8 +20,16 @@
 Debug::set($_REQUEST['debug']);
 
 if (is_numeric($_GET['device_id'])) {
-    foreach (dbFetch('SELECT * FROM ports WHERE device_id = ? ORDER BY portName,ifAlias', [$_GET['device_id']]) as $interface) {
+    // use php to sort since we need call cleanPort
+    $interface_map = [];
+    foreach (dbFetch('SELECT * FROM ports WHERE device_id = ?', [$_GET['device_id']]) as $interface) {
         $interface = cleanPort($interface);
+        $interface_map[$interface['label']] = $interface;
+    }
+    $interface_names = array_keys($interface_map);
```

Comment:
Is there any point in doing array_keys and sort instead of just ksort?

Reply:
You are right, changed